prisma/mongodb/schema.prisma

```diff
@@ -17,14 +17,16 @@ model Question {
     updatedAt DateTime @updatedAt
     title     String
     body      String
-    answers   Answer[]
+    difficulty Int
+    category  String
+    // answers   Answer[]
```

Comment:
```suggestion
    @@index([difficulty])
```
We don't need to add it now but we could consider adding a secondary index so we can look up questions by difficulty / category in future

Reply:
@samyipsh Could we do this in a new PR?